.isort.cfg

```diff
@@ -0,0 +1,15 @@
+[settings]
+combine_as_imports=1
+include_trailing_comma=1
+lines_after_imports=2
+multi_line_output=3
+skip=.tox,migrations
+not_skip=__init__.py
+use_parentheses=1
+known_django=django,haystack
+known_wagtail=wagtail
+known_future_library=future,six
+known_third_party=mock
+known_first_party=alerts,ask_cfpb,cfgov,core,data_research,jobmanager,search,v1,wellbeing
```

Comment:
I worry this might be forgotten the next time someone adds a new project. Not sure there's an easy way to have this automatically count local directories as first party?

Reply:
Without this the sorting was a bit weird sometimes, but it'd get 90% there. I can experiment a bit more.